### ISSUE TITLE
读取视频需要硬解码吗？

### ISSUE THREAD

**Issue by dszs123** [leafqycc/rknn-cpp-Multithreading#3]
我使用这个源码，编译运行后，速度只有3帧。耗时都在读取视频上。
不需要硬件解码器吗？

我用的是rk3588s

**Comment by leafqycc** [comment]
> 我使用这个源码，编译运行后，速度只有3帧。耗时都在读取视频上。
> 不需要硬件解码器吗？
> 
> 我用的是rk3588s

可以试一试，但是我这边用read软解也没问题，要不你试试换个桌面后端服务器？比如说换成xorg或者wayland(也许是不同环境下GPU的适配不同导致的？)

**Comment by dszs123** [comment]
只是capture.read上耗时了。imshow关掉了。和gpu也有关系吗？
我使用的是firefly开发板，他们官方的ubuntu系统。
香橙派的就没有这个问题。

**Comment by whatt-ai** [comment]
我这也只是这个问题，读取4fps，给我吓坏了

**Comment by YCG1** [comment]
你可以使用FFmpeg来解封装，使用MPP来解码效率很高

**Comment by Felix-fz** [comment]
> 我这也只是这个问题，读取4fps，给我吓坏了

请问您这个问题解决了吗

**Comment by Felix-fz** [comment]
> 的就没有这个问题。

请问您最终解决这个视频底的问题了吗

**Comment by YCG1** [comment]
> > 的就没有这个问题。
> 
> 请问您最终解决这个视频底的问题了吗

是的 我已经实现了 多路视频拉流 硬解码 并且高效推理 不是本仓库的多线程方案 效率跟高 资源占用更低 QQ 1281854907
